qiskit_ibm_runtime/utils/noise_learner_result.py

```diff
@@ -170,9 +178,9 @@ def qubits(self) -> list[int]:
         return self._qubits
 
     @property
-    def error(self) -> PauliLindbladError:
+    def error(self) -> Union[PauliLindbladError, None]:
         r"""
-        The error channel in this :class:`.~LayerError`.
+        The error channel in this :class:`.~LayerError`, or ``None`` if the error channel is unknown.
```

Comment:
If you want to also allow treating None as a user value as disabling maybe something like this?
```suggestion
        The error channel in this :class:`.~LayerError`, or ``None`` if the error channel is either unknown or explicitly disabled.
```